AGENTS.md

```
@@ -34,6 +34,51 @@
+### アンチパターン
+
+- 早すぎる共通化で可読性が下がる
+- 巨大全能ヘルパーの作成
+
+### 関連タグ（refactor）
+
+`refactor:dedupe`, `refactor:simplify`（定数化・命名整理）
+
+### コミット例
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)